src/App.cpp

```diff
@@ -146,8 +145,8 @@ App::ReadyToRun()
 
 	LoadPreferences();
 
-	if (fFile != NULL) {
-		_ShowMainWindow(*fFile);
+	if (fFile.InitCheck() == B_OK) {
```

Comment:
I think, you can keep != NULL :)

Reply:
BPath has operators != und == , so maybe if (fFile != NULL)  could also work.

Reply:
I tried, it does work. But since there are plenty InitCheck()s around already in the code, I'll just keep it here as well.

Reply:
OK!